doc/source/getting_started/comparison/includes/missing.rst

```diff
@@ -0,0 +1,24 @@
+This doesn't work in pandas.  Instead, the :func:`pd.isna` or :func:`pd.notna` functions
+should be used for comparisons.
+
+.. ipython:: python
+
+   outer_join[pd.isna(outer_join["value_x"])]
+   outer_join[pd.notna(outer_join["value_x"])]
+
+pandas also provides a variety of methods to work with missing data -- some of
+which would be challenging to express in Stata. For example, there are methods to
```

Comment:
Argh, missed the reference to Stata here. Fixed in https://github.com/pandas-dev/pandas/pull/38935.